Updated (json file) to new version (5.0.53)

dalanicolai opened the pull request:
Updated (json file) to new version (5.0.53)

nedrichards commented:
don't forget to update the appdata as well!

amtlib-dot-dll commented:
Strange, I have already updated it? https://github.com/flathub/org.zotero.Zotero/commit/5bcf09993a51a1ab106d4cab2b20f89c4feb769e

By the way, I wrote a [script](https://github.com/flathub/org.zotero.Zotero/blob/master/build.sh) to automatically update these files, but is there any infrastructure to run it? @nedrichards

dalanicolai commented:
Yes, that's strange. Do you understand this already? Indeed I did not sync
with the original repository before I updated the json file (I will remeber
for next time, but this time I was new to git en github). But anyway the
pull request shows that the json file in the original repository was not
updated yet (despite of your commit with the update).

Nice script, I'll try to learn from it.

Thanks for reminding me about the appdata, I'll have a look at it...

On 13 July 2018 at 18:43, 朝歌 <notifications@github.com> wrote:

> Strange, I have already updated it? 5bcf099
> <https://github.com/flathub/org.zotero.Zotero/commit/5bcf09993a51a1ab106d4cab2b20f89c4feb769e>
>
> By the way, I wrote a script
> <https://github.com/flathub/org.zotero.Zotero/blob/master/build.sh> to
> automatically update these files, but is there any infrastructure to run
> it? @nedrichards <https://github.com/nedrichards>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/flathub/org.zotero.Zotero/pull/10#issuecomment-404888288>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARk3XwdHBk78mxczGeKZUXLM4fFIdBQgks5uGM47gaJpZM4VOpoi>
> .
>
